rotary_utils/repair.py

```diff
@@ -7,14 +7,15 @@
 import logging
 import os
 import shutil
-import subprocess
 import sys
 
 import pandas as pd
 from Bio import SeqIO
 
-from rotary_utils.utils import check_dependencies, run_pipeline_subcommand, set_write_mode, load_fasta_sequences
+from rotary_utils.external import map_long_reads, subset_reads_from_bam, run_flye, run_circlator_merge, \
+    check_circlator_success, check_dependencies
 from rotary_utils.rotate import rotate_sequences_wf
```

Comment:
@jmtsuji What does the `wf` stand for?

Reply:
@jmtsuji Change the `wf` part to `workflow` or remove the `wf.` Is knowing it is a workflow critical, or is `rotate_sequences` good enough? We generally don't want to use acronyms in our variable names.

Reply:
Removed the `_wf` part -- thanks for the advice.